alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+from database import Base
+from city_app.models import *
+from temperature_app.models import *
```

Comment:
You don't need this. `from database import Base` is sufficient because it automatically includes all your models.